src/View/Helper/PaginatorHelper.php

```diff
@@ -296,7 +296,7 @@ protected function _toggledLink($text, $enabled, $options, $templates): string
         $text = $options['escape'] ? h($text) : $text;
 
         $templater = $this->templater();
-        $newTemplates = !empty($options['templates']) ? $options['templates'] : false;
+        $newTemplates = $options['templates'] ?? false;
```

Comment:
Do we care if templates are `''`?

Reply:
I made `StringTemplate::load()` an exception instead if argument is empty string.